.gitignore

```diff
@@ -0,0 +1,2 @@
+bin/
+obj/
```

Comment:
I would just save the time and merge or replace this with the contents from the [.Net-SDK's .gitignore](https://raw.githubusercontent.com/dapr/dotnet-sdk/master/.gitignore):
```suggestion

# Everything in the external/packages directory as nuget packages are restored here.
/nuget/restored_packages/*

# Build generated files in src
/src/**/objd/**/*
/src/**/obj/**/*
/src/**/bin/**/*
/src/**/gen/**/*
/src/**/StyleCop.Cache

# components generated by Dapr when run in samples project directory.
/samples/**/components/*.yaml

# launchSettings.json for samples and tests
/test/**/Properties/launchSettings.json

# Test logs
TestResults/

# MSBuild's log file
msbuild.log

# test results
TestResults/

# Build.exe
*.err
*.wrn
*.log
*.prf
*.trc
*.nupkg
buildd.dbb
build.dbb
buildd.evt
build.evt

###
Repository Bloat Risks
###
*.exe
*.dll
*.lib
*.pdb
*.wim
*.zip
*.gz
*.meta
*.obj
*.tmp
*.tmp_proj
*.log
.builds

## Ignore Visual Studio temporary files, build results, and
## files generated by popular Visual Studio add-ons.
**/.vs/**/*
/test/**/v15/Server/sqlite3/**/*

# User-specific files
*.suo
*.user
*.userosscache
*.sln.docstates

# Build results
[Dd]ebug/
[Dd]ebugPublic/
[Rr]elease/
[Rr]eleases/
x64/
x86/
build/
bld/
[Bb]in/
[Oo]bj/

#drop folder
/src/drop/**/*

# Roslyn cache directories
*.ide/

# Visual Studio profiler
*.psess
*.vsp
*.vspx
/drop

# VS Code
.vscode/

# Jetbrains
.idea/

# coverlet code coverage results
coverage.json
.fake
.ionide

# Examples bloat
examples/**/tmp
# macOS
.DS_Store
```

Reply:
Updated.